rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -730,6 +730,15 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		ui->fsrSharpeningStrength->setValue(fsr_sharpening_strength_def);
 	});
 
+	//DoubleSpinBoxes
+	m_emu_settings->EnhanceDoubleSpinBox(ui->TextureLODBias, emu_settings_type::TextureLodBias);
+	SubscribeTooltip(ui->TextureLODBias, tooltips.settings.texture_lod_bias);
+	ui->TextureLODBias->setRange(-16, 16);
+	connect(ui->TextureLODBiasReset, &QAbstractButton::clicked, [this]()
+	{
+		ui->TextureLODBias->setValue(0);
```

Comment:
This needs to set the value to the default, and not some magic number.

Reply:
```suggestion
		ui->TextureLODBias->setValue(stod(m_emu_settings->GetSettingDefault(emu_settings_type::TextureLodBias)));
```

Reply:
That looks nicer I'll try that